README.md

```diff
@@ -24,6 +24,424 @@ Plugin for managing GithubAction files in [Leto-Modelizer](https://github.com/di
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">discussion_comment</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">fork</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">gollum</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">issue_comment</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">issues</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">label</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">merge_group</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">milestone</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">page_build</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">project</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">project_card</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">project_column</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">public</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">pull_request</td>
+      <td align="center">$\textcolor{orange}{\textsf{〜}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td><td width="5px" padding="0"></td>
```

Comment:
divide (for under repetition too)

Reply:
no sorry, is more easy to edit it like that.